scram/route_manager/admin.py

```diff
@@ -14,7 +14,14 @@ class ActionTypeAdmin(SimpleHistoryAdmin):
     list_display = ("name", "available")
 
 
-admin.site.register(Entry, SimpleHistoryAdmin)
+@admin.register(Entry)
+class EntryAdmin(SimpleHistoryAdmin):
+    """Configure how Entries show up in the Admin site."""
+
+    list_filter = ("is_active", "who")
```

Comment:
I think we should change this to:

```
list_filter = [
    "is_active",
    ("who", admin.RelatedOnlyFieldListFilter),
]
```

That will only return users who have actually blocked a thing (or blocked a thing that meets the other filter criteria).

Reply:
That sounds like a useful change, thanks.